src/transformers/pipelines/__init__.py

```diff
@@ -441,6 +443,12 @@ def check_task(task: str) -> Tuple[str, Dict, Any]:
             - `"summarization"`
             - `"zero-shot-classification"`
             - `"zero-shot-image-classification"`
+            - `"zero-shot-object-detection"`
+            - `"visual-question-answering"`
+            - `"document-question-answering"`
+            - `"object-detection"`
+            - `"depth-estimation"`
+            - `"video-classification"`
```

Comment:
Would be goo to have those alphabetically sorted, like the rest.

Reply:
Thanks, will do!